src/js/components/Chart/doc.js

```diff
@@ -176,15 +176,24 @@ const data = calcs(<values>, { coarseness: 5, steps: [1, 1] });`,
 };
 
 export const themeDoc = {
+  'chart.color': {
```

Comment:
Let's add `color` to Chart typing, here https://github.com/grommet/grommet/blob/e6bac4512e0ae18fdfce17ba65f4a7d920f1000a/src/js/themes/base.d.ts#L319

Reply:
done